include/opengl/OpenGlDef.h

```diff
@@ -33,6 +33,12 @@
 
 #endif
 
+#elif defined(__arch64__) && defined(__linux__)
```

Comment:
I think doesnt' working it's aarch64 defined and not arch64 typo

Reply:
Would make sense. @Specht2010 does that seem sensible?

Reply:
Ah yes that's definitely a typo, sorry.  It should be aarch64.

Reply:
Thanks! Made the change. https://github.com/jpd002/Play--Framework/commit/f9209d7d52df1b91976287256af60b00a38b26ae